### PR TITLE
Add ER quota description in quota doc

### PR DESCRIPTION
**Pull request by lichuqiang**
ref: https://github.com/kubernetes/kubernetes/pull/57302

### CODE REVIEW

File: docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -64,6 +64,23 @@ The following resource types are supported:
 | `requests.cpu` | Across all pods in a non-terminal state, the sum of CPU requests cannot exceed this value. |
 | `requests.memory` | Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value. |
 
+### Resource Quota For Extended Resources
+
+In addition to the resources mentioned above, in release 1.10, quota support for
+[extended resources](/docs/concepts/configuration/manage-compute-resources-container.md#extended-resources) is added.
+
+As overcommit is not allowed for extended resources, it makes no sense to specify both `requests`
+and `limits` for a same extended resource in a quota. So for extended resources, only quota items
```
Comment (tengqm) [inline docs/concepts/policy/resource-quotas.md]: > a same extended

the same extended

File: docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -64,6 +64,23 @@ The following resource types are supported:
...
+with prefix `requests.` is allowed for now.
+
+Take the GPU resource as an example, if the resource name is `nvidia.com/gpu`, and you want to
+limit the total sum of GPU in a namespace to "4", then you can define a quota as follows:
```
Comment (tengqm) [inline docs/concepts/policy/resource-quotas.md]: > total sum of GPU

total number of GPUs requested

Comment (tengqm) [reply docs/concepts/policy/resource-quotas.md]: remove "then"

Comment (tengqm) [reply docs/concepts/policy/resource-quotas.md]: We can safely remove the quotation marks around 4, right?

**Comment by heckj** [comment]
@lichuqiang please make the wording updates that @tengqm has suggested

/assign

**Comment by lichuqiang** [comment]
@heckj @tengqm Done :)

**Pull request merged**